app/views/games/_team_list.html.haml

```diff
@@ -13,13 +14,14 @@
       / Get the first teams score and use that as the full score bar
       - max_score = teams.first.current_score
       / We pass in starting_rank here to start the list at the proper index
-      = render partial: 'games/team_list_tbody', locals: { teams: teams.slice(0..4), starting_rank: 1, max_score: max_score }
+      = render partial: 'games/team_list_tbody', locals: { teams: teams.slice(0..5), starting_rank: 1, max_score: max_score, tbody_id: '' }
```

Comment:
Setting the `tbody_id` with a default value in the `games/team_list_tbody` avoids the `tbody_id` needing to be set to `''` here.